README.md

```diff
@@ -50,80 +50,108 @@ The DeepSparse Engine is a CPU runtime that delivers unprecedented performance b
+Here is how to quickly perform inference with DeepSparse Engine on a pre-trained dense ResNet-50 from SparseZoo.
 
 ```python
 from deepsparse import compile_model
 from sparsezoo.models import classification
+
 batch_size = 64
 
 # Download model and compile as optimized executable for your machine
-model = classification.mobilenet_v1()
+model = classification.resnet_50()
 engine = compile_model(model, batch_size=batch_size)
 
 # Fetch sample input and predict output using engine
 inputs = model.data_inputs.sample_batch(batch_size=batch_size)
 outputs, inference_time = engine.timed_run(inputs)
 ```
 
-**MobileNetV1 Optimized**
+**ResNet-50 Sparsified**
 
 When exploring available optimized models, you can use the `Zoo.search_optimized_models` utility to find models that share a base.
 
-Let us try this on the dense MobileNetV1 to see what is available.
+Let us try this on the dense ResNet-50 to see what is available.
 
 ```python
 from sparsezoo import Zoo
 from sparsezoo.models import classification
-print(Zoo.search_optimized_models(classification.mobilenet_v1()))
+
+model = classification.resnet_50()
+print(Zoo.search_optimized_models(model))
 ```
 
 Output:
 
 ```shell
-[Model(stub=cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/base-none),
- Model(stub=cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-conservative),
- Model(stub=cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate),
- Model(stub=cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned_quant-moderate)]
+[
+    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/base-none), 
+    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-conservative), 
+    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate), 
+    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned_quant-moderate), 
+    Model(stub=cv/classification/resnet_v1-50/pytorch/sparseml/imagenet-augmented/pruned_quant-aggressive)
+]
 ```
 
-Great. We can see there are two pruned versions targeting FP32, `conservative` at 100% and `moderate` at >= 99% of baseline accuracy. There is also a `pruned_quant` variant targetting INT8.
+We can see there are two pruned versions targeting FP32 and two pruned, quantized versions targeting INT8.
+The `conservative`, `moderate`, and `aggressive` tags recover to 100%, >=99%, and <99% of baseline accuracy respectively.
 
-Let's say you want to evaluate best performance on FP32 and are okay with a small drop in accuracy, so we can choose `pruned-moderate` over `pruned-conservative`.
+Let's say that we want something that recovers close to the baseline and is very performant, we can choose the pruned_quant-moderate model.
+This model will run [nearly 7 times faster](linktoresnet50example) than the baseline model on a compatible CPU (VNNI instruction set enabled).
```

Comment:
placeholder link: `[nearly 7 times faster](linktoresnet50example) `

Reply:
waiting on the link to be provided to include here